backend/seo_api.py

```
@@ -0,0 +1,288 @@
+        })
+
+    log.info("Delegated %d SEO work package(s) from audit %s", len(created), audit_id)
+    return SeoDelegationCreateResult(
+        audit_id=audit_id, created=len(created), tasks=created,
+    )
+
+
+@router.post("/company/{company_id}/seo/fix", response_model=SeoFixResult)
+async def run_seo_fixes(
+    company_id: str = Path(..., description="Company ID"),
+    request: SeoFixRequest = Body(...),
```

Ruff (0.15.15)

[warning] Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)